client/src/components/Reviews/components/Review.jsx

```diff
@@ -1,14 +1,65 @@
 // may add a diffrent file for helfult/report buttons
 import React from 'react';
 import styled from 'styled-components';
+import { formatDistanceToNow, parseISO } from 'date-fns';
 
 const StyledReview = styled.div`
   border: 2px solid black;
   margin: 10px;
 `;
+const Stars = styled.p`
+  padding-top: 10px;
+  float: left;
+  padding-left: 10px;
+`;
+const User = styled.p`
+  padding-top: 10px;
+  float: right;
+  padding-right: 10px;
+`;
+const Title = styled.h2`
+  padding-top: 10px;
+  font-size: 150%;
+`;
+const Buttons = styled.button`
+  background: none!important;
+  border: none;
+  cursor: pointer;
+`;
 
-function Review() {
-  return <StyledReview>review</StyledReview>;
+function Review({ review }) {
+  let date;
+  let rec = '';
+  function HandleReviewClick(e, type) {
+    e.preventDefault();
+    // send request to review.review_id, type (helpful or report)
+  }
+  function ParseData() {
+    date = formatDistanceToNow(parseISO(review.date));
+    if (review.recommend) {
+      rec = '✅';
+    }
+  }
+  ParseData();
+  return (
+    <StyledReview>
+      <Stars>{review.rating}</Stars>
+      {/* need to add fn to display stars */}
+      <User>
```

Comment:
Are these going to be React components? If so are there any issues with not making them self closing elements? I've only seen React components of the form
```
<Component prop1={...}, prop2={...}, ... />
```

Reply:
Nevermind, I see now that they're styled components.

Reply:
Up to you. I don't think it's strictly necessary though. I just wasn't familiar with the styled components syntax.